website/docs/r/vapp_vm.html.markdown

```diff
@@ -113,6 +113,29 @@ resource "vcd_vapp_vm" "web2" {
 
   depends_on = ["vcd_vapp.web"]
 }
+```
+
+## Example Usage (Override Template Disk)
+
```

Comment:
I suggest adding a sentence here:
"This example shows how to [change VM template's disk properties](link to Override template Disk section) when the VM is created."

Reply:
done